bestfirstsearch.java

```diff
@@ -169,7 +177,7 @@ public void bestfirstsearch()
 System.out.print(arr1[i]+"\t");
 }
 int cost=0;
-int v=r-1;
+@LTLengthOf("arr") int v=r-1;
```

Comment:
As in many other cases, please consider using `@IndexFor` instead of `@LtLengthOf`.

Reply:
okay done!